src/apps/VirtualApp/VirtualChat.js

```diff
@@ -265,20 +320,36 @@ class VirtualChat extends Component {
       );
     });
 
+    let questions_msgs = questions.map((msg, i) => {
```

Comment:
Should probably be ordered from oldest on top to newest on bottom, see the other chats...